src/Umbraco.Web.UI.Client/src/common/directives/components/editor/umbeditornavigationitem.directive.js

```diff
@@ -0,0 +1,51 @@
+/**
+@ngdoc directive
+@name umbraco.directives.directive:umbTabContent
```

Comment:
@nielslyngsoe the eagle-eyed nit-pick is that `umbTabContent` is no longer named this and the angular JS docs we generate will be wrong & mis-leading